test/integration/charge_ft_tests.js

```diff
@@ -454,7 +454,7 @@ describe('chargeTests', function () {
           .set('Content-Type', 'application/x-www-form-urlencoded')
           .set('Cookie', ['frontend_state=' + cookieValue])
           .set('Accept', 'application/json')
-          .expect(500)
+          .expect(200)
```

Comment:
Why has this changed?

Reply:
because `enter_card_details` is no longer in a terminal state, therefore it's not a 500 response.